File: s3/cycle1/java/ex04.java
import java.util.*;
public class ex04 {
	public static void main(String[] args) {
		int i,n;
		char[] str , revStr = new char[40];
		System.out.print("Enter the string: ");
		Scanner input = new Scanner(System.in);
		String s = input.nextLine();
		n = s.length();
		str = s.toCharArray();
		for( i = 0; i < n ; i++ )
			revStr[i] = str[n-i-1];
		String revS = new String(revStr);
		System.out.println("Reversed String : " + revS);
	}
}

File: s3/cycle1/ex04.c
#include <stdio.h>
#include <string.h>

int main(void)
{
	char s[50] , revS[50];
	int i,n;
	printf("Enter the string: ");
	scanf("%s" , s);
	n = strlen(s);
	for(i = 0 ; i < n ; i++)
		revS[i] = s[n - 1 -i];
	printf("\nReversed : %s\n", revS);
}